frontend/src/components/TextEditor.jsx

```diff
@@ -26,18 +26,16 @@ export default function TextEditor({ document = [], onChange }) {
   }, [editor, addCommentThread]);
 
   return (
-    <div className={"flex flex-col"}>
+    <div className="flex flex-col h-full">
       <Slate editor={editor} initialValue={document} onChange={onChange}>
         <Toolbar />
         <div className="bg-document-background flex flex-row justify-center">
-          <div
-            className={"bg-white mx-40 mt-20 mb-7 p-16 max-w-4xl min-h-screen "}
-          >
+          <div className="bg-white mx-40 mt-20 mb-7  max-w-4xl min-h-screen">
```

Comment:
![image](https://github.com/wendi-yu/newbloom/assets/44381949/e0b3ad1b-b5c5-42f7-960b-b5c5ddbc65a8)
:o i think put padding back

Reply:
wait idk why its just in document view